src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -153,6 +153,7 @@ private static String convertElementNode(Node currentNode, String indentation, b
                 Node attribute = attributes.item(i);
                 if (isTooltipAttribute(attribute)
                         || isPopoverAttribute(attribute)
+                        || isHandsontableAttribute(attribute) // ignored due to
```

Comment:
Unfinished comment: due to what? Btw, what's the purpose of this `convertElementNode` and why do you need to ignore it being applied to the spreadsheet `div`?

Reply:
This was done to accommodate issues with screen resolution. `InstructorCourseEnrollPageUiTest` was passing in my local build but failed in the remote Travis. I checked the failure and it was due to the difference in pixel values. Took the hint from this [comment](https://github.com/TEAMMATES/teammates/pull/7568#issuecomment-314664360).